salt/recommendations/config/srv-recommendations-containers.env

```diff
@@ -0,0 +1,9 @@
+{%- if pillar.elife.env == 'dev' -%}
+{%- set app_env = 'ci' -%}
```

Comment:
Why's this needed?

Reply:
https://github.com/elifesciences/recommendations/blob/develop/web/app.php#L10 loads `WebProfilerServiceProvider` which is a dev dependency and isn't in the published image. In these cases Vagrant isn't really a development environment anymore as code can't be edited within it; only a local replica of AWS servers.

Reply:
Maybe we should use `continuumtest`. CI-related setup won't be published either (assume there's none in this case though).

Reply:
It rings a bell. However `ci` is still the [default for ad-hoc stacks](https://github.com/elifesciences/builder-private/blob/master/pillar/elife.sls#L11) such as the ones used to build formulas or [created to do a demo](https://github.com/elifesciences/builder-private/blob/master/pillar/elife.sls#L11), so I'd keep it consistent with that.